pkg/reader/ccip.go

```diff
@@ -1054,8 +1059,8 @@ func (r *ccipChainReader) getFeeQuoterTokenPriceUSD(ctx context.Context, tokenAd
 type sourceChainConfig struct {
 	Router    []byte // local router
 	IsEnabled bool
-	OnRamp    []byte
 	MinSeqNr  uint64
+	OnRamp    []byte
```

Comment:
Seems like I missed these
```suggestion
	OnRamp    UnknownAddress
```